app/models/concerns/compute_scaling.rb

```diff
@@ -0,0 +1,35 @@
+# dynamic GCE instance scaling based on input file size
+module ComputeScaling
+  extend ActiveSupport::Concern
+
+  # GCE machine types and file size ranges
+  # produces a hash with entries like { 'n2-highmem-4' => 0..16.gigabytes }
+  # change GB_PER_CORE in parent class to adjust scaling (1-8, lower numbers means more aggressive scaling)
```

Comment:
Increasing GB_PER_CORE increases memory per core, which seems like _more_ (not less) aggressive scaling, right?

```suggestion
  # change GB_PER_CORE in parent class to adjust scaling (1-8, higher number means more aggressive scaling)
```

Reply:
Ah yes - you are correct.  Great catch!